vsts-arm-test-steps.yml

```diff
@@ -53,6 +53,15 @@ steps:
   env:
     CIRCLE_TOKEN: $(CIRCLECI_TOKEN)
 
+- bash: |
+   export NATIVE_UNITTESTS_DEST=$PWD/src/out/Default
+   cd src/electron
+   node script/download-circleci-artifacts.js --buildNum=$CIRCLE_BUILD_NUM --name=shell_browser_ui_unittests --dest=$NATIVE_UNITTESTS_DEST
+   chmod +x $NATIVE_UNITTESTS_DEST/shell_browser_ui_unittests
```

Comment:
it's surprising to me that we have to set +x here...

Reply:
me too - for some reason in testing the version that gets pulled down isn't executable 🤔